function.go

```diff
@@ -100,6 +100,22 @@ type Function struct {
 	// according to the client which is in charge of what constitutes being
 	// fully "Created" (aka initialized)
 	Created time.Time
+
+	// Invocation defines hints for use when invoking this function.
+	// See Client.Invoke for usage.
+	Invocation Invocation `yaml:"invocation,omitempty"`
```

Comment:
Ahh now I understand what "Created prior" means.

Reply:
Ay, that was _thoughtus interruptus_ indeed.  We do have an edge case which is backwards incompatible.

Reply:
> We do have an edge case which is backwards incompatible.

We're still pre-1.0, we can do anything! bwahahahah! :wink: